noark-extraction-validator/pom.xml

```diff
@@ -28,22 +28,21 @@
 			<optional>true</optional>
 		</dependency>
 
+		<!-- XJC (XSD schema converter) -->
+
 		<dependency>
-			<groupId>commons-logging</groupId>
```

Comment:
This package is needed for running the project as a standalone jar tool.

Reply:
Addressed w/ https://github.com/documaster/noark-extraction-validator/pull/66/commits/b94c3fa94913345f901788b827972196eca60618